docs/v3/array-storage-transformers.rst

```diff
@@ -1,13 +1,29 @@
+.. _storage-transformers-list:
+
 ==========================
 Array Storage Transformers
 ==========================
 
-Under construction.
+.. COMMENT TO BE REMOVED WHEN ONE IS ADDED
+
+   The following documents specify transformers which are defined by the maintainers of
+   the Zarr specification. Being listed below does not imply that a transformer is
+   required to be implemented by implementations.
 
-.. toctree::
+   toctree::
    :glob:
    :maxdepth: 1
    :titlesonly:
    :caption: Contents:
 
    array-storage-transformers/*/*
+
+Currently, no core storage transformers are defined by the maintainers of
```

Comment:
why do we refer to the maintainers here? isn't it sufficient to state that the spec doesn't define any storage transformers?

Reply:
Pushed a simpler version and we can keep simplifying it.